tests/browserstack/click-lookup.test.js

```diff
@@ -1,18 +1,24 @@
 
 describe('click-lookup.test.js', () => {
-  const config = require('./config/click-lookup-config.js')
-  const alph_tests = require('../../src/alph-selenium-test-cases')
-  const configurator = require('../../src/alph-config')
-  const versions = configurator.versions(config.env, 'desktop')
+  const config = require('@tests/browserstack/config/click-lookup-config.js')
+  const alph_tests = require('@src/alph-selenium-test-cases')
+  const configurator = require('@src/alph-config')
+
+  const alph = require('@src/alph-selenium')
+  const configMain = require('@src/main-config.json')
+
+  const versionsDesktop = configurator.versions(config.env.desktop, 'desktop')
+  const versionsMobile = configurator.versions(config.env.mobile, 'mobile')
+
   const testUrl = configurator.testUrl(config)
 
   beforeEach(() => {})
   afterEach(() => {})
 
   // console.info('versions', versions)
-  versions.forEach(version => {
-    it(`should execute click lookup - ${version.name}`, async () => {
-      const capabilities = Object.assign(version, { buildName: 'Click lookup' } )
+  versionsDesktop.forEach(version => {
+    it.skip(`should execute click lookup - ${version.name}`, async () => {

```

Comment:
should this remain skipped?

Reply:
I need it only for limiting current testing for mobile.
all desktop versions are already checked (including all last changes for mobile testing)